vestapol/destinations/gcp_destination.py

```diff
@@ -57,6 +58,15 @@ def create_table(self, resource: ResourceTypes):
 
         tablename = resource.name
 
+        if resource.manual_schema is not None:
```

Comment:
Something to consider: the two "public" methods for Vestapol are `resource.load(destination)` and `destination.create_table(resource)`, but `resource.load(destination)` doesn't need to know about any sort of schema since it's just extracting data. If you instead passed the `manual_schema` in as an optional parameter to `create_table()`, it reduces the surface area of this change a bit—you wouldn't need to make any changes to `base_resource.py` or `json_resource.py`.

Either way works though.